etc/viam-csi-pi-arm64.yml

```
@@ -0,0 +1,55 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+script:
+- rm -rf $TARGET_APPDIR | true
```

```suggestion
- rm -rf $TARGET_APPDIR || true
```
Unless you meant to pipe for some reason?

---

thanks!